README.md

```diff
@@ -143,8 +143,10 @@ customModules:
             path: /academy/1-what-is-cosmos/1-blockchain-and-cosmos.html
           - title: The Interchain Ecosystem
             path: /academy/1-what-is-cosmos/2-cosmos-ecosystem.html
+          - title: Interchain use cases
+            path: /academy/1-what-is-cosmos/3-interchain-use-cases.html
           - title: Getting ATOM and staking it
-            path: /academy/1-what-is-cosmos/3-atom-staking.html
+            path: /academy/1-what-is-cosmos/4-atom-staking.html

```

Comment:
We should make sure to check all internal links - no more `3-atom-staking`.

Reply:
Quickly gone through all the W1 content - updated a couple of links that were commented out just to be safe.